src/java.base/share/classes/java/net/URLEncoder.java

```diff
@@ -293,7 +293,23 @@ public static String encode(String s, Charset charset) {
         return out.toString();
     }
 
-    private static void flushToStringBuilder(StringBuilder out, CharsetEncoder ce, CharBuffer cb, ByteBuffer bb, boolean endOfInput) {
+    /**
+     * Encodes input chars in cb and appends the byte values in an escaped format
+     * ("%FF") to out. The temporary byte buffer, bb, must be able to accept
+     * cb.position() * ce.maxBytesPerChar() bytes.
```

Comment:
```suggestion
     * Encodes input chars in {@code cb} and appends the byte values in an escaped
     * format ({@code "%XX"}) to {@code out}. The temporary byte buffer, {@code bb},
     * must be able to accept {@code cb.position() * ce.maxBytesPerChar()} bytes.
```

Reply:
As this javadoc isn't published anywhere I didn't bother with `{@code..`, but since you've spent time to fix things up then why not. 👍🏽